workers/unity/Packages/com.improbable.gdk.core/Utility/SnapshotBuilder.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class SnapshotBuilder
```

Comment:
Given this class doesn't actually follow the Builder pattern - can we rename this?